MAINTAINERS

```diff
@@ -588,3 +590,7 @@ made through a pull request.
 	Name = "Vishnu Kannan"
 	Email = "vishnuk@google.com"
 	GitHub = "vishh"
+
+	[people.mary]
+	Name = "Mary Anthony"
+	Email = "mary.anthony@docker.com"
```

Comment:
I already had a patch fixing something in it with tomlv too, so I just
added it to that, pinged you on it too :)

On Thu, Feb 19, 2015 at 11:45 AM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In MAINTAINERS
> <https://github.com/docker/docker/pull/10866#discussion_r25018358>:
>
> > @@ -588,3 +590,7 @@ made through a pull request.
> >  	Name = "Vishnu Kannan"
> >  	Email = "vishnuk@google.com"
> >  	GitHub = "vishh"
> > +
> > +	[people.mary]
> > +	Name = "Mary Anthony"
> > +	Email = "mary.anthony@docker.com"
>
> @jfrazelle <https://github.com/jfrazelle> Want me to do that? or would it
> be faster if you did it? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/10866/files#r25018358>.
>